frontend/pages/hosts/details/cards/About/About.tsx

```diff
@@ -21,6 +23,41 @@ const About = ({
   munki,
   mdm,
 }: IAboutProps): JSX.Element => {
+  const renderPublicIp = () => {
+    if (aboutData.public_ip !== "---") {
+      return aboutData.public_ip;
+    }
+    return (
+      <>
+        <span
+          className="text-cell text-muted tooltip"
+          data-tip
+          data-for={"public-ip-tooltip"}
+        >
+          {aboutData.public_ip}
+        </span>
+        <ReactTooltip
+          place="bottom"
```

Comment:
same position thing here

Reply:
This one they (Zay in CX meeting today) said they wanted on bottom because it'll cover the header if it's on top